__tests__/Lotto.test.js

```diff
@@ -0,0 +1,30 @@
+import ERROR_MESSAGE from '../src/constants/ErrorMessage.js';
+import Lotto from '../src/model/Lotto.js';
+import checkNumberInRange from '../src/Validation/checkNumberInRange.js';
+
+describe('Lotto class test', () => {
+  it('로또를 생성했을 때 6개의 번호가 아닐 경우 에러를 던진다.', () => {
+    const testCase = [1, 2, 3, 4, 5];
+    expect(() => {
+      new Lotto(testCase);
+    }).toThrow(ERROR_MESSAGE.notSixNumbers);
+  });
+
+  it('로또를 생성했을 때 중복된 숫자가 포함된 경우 에러를 던진다.', () => {
+    const testCase = [1, 2, 3, 4, 5, 5];
+    expect(() => {
+      new Lotto(testCase);
+    }).toThrow(ERROR_MESSAGE.duplicatedNumbers);
+  });
+  it('로또 생성했을 때 1-45 외의 숫자가 들어갔을때, 에러를 던진다', () => {
+    const testCase = [45, 1, 2, 3, 4, 46];
+    expect(() => {
+      checkNumberInRange(testCase);
+    }).toThrow(ERROR_MESSAGE.numberOutOfRange);
+  });
```

Comment:
여기는 같은 Context를 공유하고 있어 아래처럼 한번 더 묶을 수 있을거 같아요
```suggestion
describe('Lotto class test', () => {
  describe("로또를 생성했을 때", () => {
    it('6개의 번호가 아닐 경우 에러를 던진다.', () => {})
    it('중복된 숫자가 포함된 경우 에러를 던진다.', () => {})
    it('1-45 외의 숫자가 들어갔을때, 에러를 던진다', () => {})
  })
```

Reply:
각각의 에러도 특정한 에러이니 테스트 명에 명시 되면 좋을거 같네요!